cmd/kwil-admin/cmds/validators/list.go

```diff
@@ -16,7 +16,7 @@ var (
 	listLong = `List the current validator set of the network.`
 
 	listExample = `# List the current validator set of the network
-kwild validators list`
+kwil-admin validators list-validators`
```

Comment:
Ah, `kwild` here is totally a typo, but we have since aliased `list-validators` to `list` since it's already a subcommand of `validators` command.  Can you verify?

Reply:
Ah right, it looks like both commands work. Will update to use the alias.